README.md

```diff
@@ -37,10 +55,19 @@ Install all javascript components using bower
 $ bower update
 ``
 
+``
+$ ionic io init
+``
+
+``
+$ ionic state reset
```

Comment:
I don't think you need reset while installing from a fresh repo (I know you had to do reset but I think that was the obsolete upstream). 

I think that the general rule of thumb is:
- fresh install (`ionic state restore`)
- pull into existing repo when plugins have changed (`ionic state reset`)